media/com_templates/css/admin-templates-default.css

```diff
@@ -70,10 +70,15 @@
 
 #jform_show_core {
 	display: inline;
-	margin-right: 100px;
+	margin-right: 7em;
 }
 
 #jform_show_diff {
 	display: inline;
-	margin-right: 50px;
+	margin-right: 5.3em;
+}
+
+.switcher-label-0, .switcher-label-1{
```

Comment:
this will not work with the new switcher as the new code is a true web component and we cannot penetrate in the shadow dom. Just add the required width of the parent (joomla-field-switcher) element, everything else will be handled in the element itself

Reply:
I do not understand exactly what you mean. If I change the width in the element, the width of the span in which the text is located does not change.

![bildschirmfoto vom 2018-07-20 10-57-54](https://user-images.githubusercontent.com/9974686/42993562-f6a8773a-8c0b-11e8-8e84-5a3267b10c73.png)

Only the width of the switcher changes.

![bildschirmfoto vom 2018-07-20 11-00-32](https://user-images.githubusercontent.com/9974686/42993651-314b49da-8c0c-11e8-963b-802a1ae1871a.png)

Reply:
The switcher will become a web component: https://github.com/joomla/40-backend-template/pull/395
That means your css will have zero effect on the span of the joomla-field-switcher ;)

Reply:
Please open an issue in the main repo about the width of the switcher, don't patch things here!

Reply:
Issue opened: https://github.com/joomla/joomla-cms/issues/21198

Reply:
I leave the CSS like that now. In the case that this PR will be merged, I will open an issue so we can rework when https://github.com/joomla/40-backend-template/pull/395 gemerged. OK?